clique_wp.sh

```diff
@@ -224,31 +215,26 @@ fi
 
 # Clean up some root installation files
 echo -e "${VP_RED}Deleting ${VP_BOLD}import.sql${VP_NONE} and the ${VP_BOLD}duplicator.zip${VP_NONE} files.${VP_WHITE}"
-rm -rf ./*.zip
-rm -rf import.sql
-rm -rf CLIQUE-CHANGELOG.txt
-rm -rf admin-creds-PLEASE-DELETE.txt
+rm -rf ./*.zip import.sql CLIQUE-CHANGELOG.txt admin-creds-PLEASE-DELETE.txt
```

Comment:
`rm` can take multiple arguments. In this case it's better to only place one call to `rm`.